design_docs/db/schema/conversations.yml

```
@@ -0,0 +1,37 @@
+# Schema for the MongoDB collection that stores ratings
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+properties:
+  messages:
+    type: array
+    items:
+      type: object
+      properties:
+        role:
```

```suggestion
      properties:
        id:
          type: string
        role:
```

We'll use `Message.id` to work with individual messages in the frontend.